tests/search.spec.ts

```
@@ -6,8 +6,9 @@ type testSearchApiOptions = {
   searchApi: Api;
   request: APIRequestContext;
   query: string;
-  expectedResponse: {};
+  expectedResponseBody: {} | string;
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)